parquet-hadoop/src/main/java/parquet/filter2/statisticslevel/StatisticsFilter.java

```diff
@@ -81,6 +81,16 @@ private boolean hasNulls(ColumnChunkMetaData column) {
     Column<T> filterColumn = eq.getColumn();
     T value = eq.getValue();
     ColumnChunkMetaData columnChunk = getColumnChunk(filterColumn.getColumnPath());
+    Statistics<T> stats = columnChunk.getStatistics();
+
+    if (stats.isEmpty()) {
+     // Do not filter in case of empty statistics
+     // pre-statistics files will have no such info
+     // best not to filter these.
+     // Also, in case of all nulls in a column, empty statistics object 
+     // is returned. This fixes the read path because of that bug
+      return false;
```

Comment:
since this comment is repeated, and we are going to fix the bug in the write path, lets just make this comment something like:
`// we have no statistics available, we cannot drop any chunks`